### DISCUSSION TITLE
WSA Sideloader v1.3.9

### DISCUSSION THREAD

**Discussion by infinitepower18** [infinitepower18/WSA-Sideloader#55] [Announcements]
- Added error info from adb to help troubleshoot APK installation failures
- Updated file association icon

<hr /><em>This discussion was created from the release <a href='https://github.com/infinitepower18/WSA-Sideloader/releases/tag/v1.3.9'>WSA Sideloader v1.3.9</a>.</em>

**Comment by alnybahy40** [comment]
> * Added error info from adb to help troubleshoot APK installation failures
> * Updated file association icon
> 
> _This discussion was created from the release [WSA Sideloader v1.3.9](https://github.com/infinitepower18/WSA-Sideloader/releases/tag/v1.3.9)._
